c2corg_ui/templates/route/filters.html

```diff
@@ -37,6 +38,18 @@
     <div class="row collapse" id="moreFilters">
       ${simple_filters()}
 
+      ## Slacklining
+      <div ng-show="filtersCtrl.filters.act.length && filtersCtrl.filters.act.indexOf('slacklining') !== -1">
+        <div class="col-xs-12 col-sm-6 filter">
+          <label translate>slackline_type</label><br>
+          ${add_multiselect('sltyp', slackline_types)}
+        </div>
+        <div class="col-xs-12 col-sm-6 filter">
+          <label translate>Length</label><br>
+          <app-slider filter="rlen" filters-list="filtersCtrl.filters" max="2000" step="10" unit="m"></app-slider>
```

Comment:
2000m? Wow :)
Are there really 2000m-long lines or are you just anticipating the records coming soon?